client/app/bundles/course/assessment/question/voice-responses/NewVoicePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { toast } from 'react-toastify';
+import {
+  VoiceData,
+  VoiceFormData,
+} from 'types/course/assessment/question/voice-responses';
+
+import LoadingIndicator from 'lib/components/core/LoadingIndicator';
+import Preload from 'lib/components/wrappers/Preload';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import translations from '../../translations';
+
+import { createVoiceQuestion, fetchNewVoiceQuestion } from './operations';
+import VoiceForm from './VoiceForm';
+
+const NEW_VOICE_TEMPLATE: VoiceData['question'] = {
+  title: '',
```

Comment:
You can abstract all these common fields out

Reply:
done!!